CHANGELOG.md

```diff
@@ -55,6 +55,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 #### Logic Database
 
+- Added: Videos added to the logic database.
```

Comment:
```suggestion
- Added: 3 Videos added to the logic database.
```

Reply:
Is it worth evening adding a log for this? I know it was done it the past but does this really add anything? (also why I excluded the count)

Reply:
in this case i would probably just do it to have consistency with the past.

Count is nice to have to a) not have it be misinterpreted as "there were no videos before, now there are" (although you could also just rephrase it to get around it) and b) give an approximation on how useful it is. E.g. seeing `240 videos added` will probably give you more incentive to look into the database for videos again than `2 videos added`

Reply:
Since our videos also show up on randovania.org, it also helps there too ig.